app/internal/wm/os_android.go

```diff
@@ -728,6 +753,24 @@ func setCursor(env *C.JNIEnv, view C.jobject, name pointer.CursorName) {
 	callVoidMethod(env, view, gioView.setCursor, jvalue(curID))
 }
 
+func setStatusColor(env *C.JNIEnv, view C.jobject, color color.NRGBA) {
+	callVoidMethod(env, view, gioView.setStatusColor,
+		jvalue(uint32(color.A)<<24|uint32(color.R)<<16|uint32(color.G)<<8|uint32(color.B)),
+		jvalue(luminosity(color)),
+	)
+}
+
+func setNavigationColor(env *C.JNIEnv, view C.jobject, color color.NRGBA) {
+	callVoidMethod(env, view, gioView.setNavigationColor,
+		jvalue(uint32(color.A)<<24|uint32(color.R)<<16|uint32(color.G)<<8|uint32(color.B)),
+		jvalue(luminosity(color)),
+	)
+}
+
+func luminosity(color color.NRGBA) int64 {
```

Comment:
Why int64 specifically? Why isn't `int` good enough?

Reply:
I used int64 because jvalue is int64. I switch to `int`.